Bank_Churn/projects/bank_churn_project_1.html

```diff
@@ -0,0 +1,112 @@
+<html>
+<head>
+  <script src='https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js'></script>
+  <script type='text/javascript'>
+    var FIELDS = [
+      {
+        label: 'CreditScore:',
+        type: 'text',
+        name: 'CreditScore',
+        value: '619',
+      },
+      {
+        label: 'Age:',
+        type: 'number',
+        name: 'Age',
+        value: '42',
+      },
+      {
+        label: 'Tenure:',
+        type: 'number',
+        name: 'Tenure',
+        value: '2',
+      },
+      {
+        label: 'Balance:',
+        type: 'number',
+        name: 'Balance',
+        value: '0',
+      },
+      {
+        label: 'NumOfProducts:',
+        type: 'number',
+        name: 'NumOfProducts',
+        value: '1',
+      },
+      {
+        label: 'Has Credit Card (0 = no, 1 = yes):',
+        type: 'number',
+        name: 'HasCrCard',
+        value: '1',
+      },
+      {
+        label: 'Active Member (0 = no, 1 = yes):',
+        type: 'number',
+        name: 'IsActiveMember',
+        value: '1',
+      },
+      {
+        label: 'EstimatedSalary:',
+        type: 'number',
+        name: 'EstimatedSalary',
+        value: '101348.88',
+      },
+    ];
+
+    $(document).ready(function () {
+      function makeField(field) {
+        return `
+          ${field.label} <br />
+          <input type=${field.type} name=${field.name} value=${field.value} />
+          <br />`;
+      }
+
+      FIELDS.forEach(field => $('#myform').append(makeField(field)));
+      $('body').append(`<p id="prediction">Prediction: <p>`);
+      $('body').append(`<input type="button" value="Predict" id="PredictButton">`);
+
+      function updatePrediction(result) {
+        result = JSON.parse(result['body']);
+        var innerHTML = $('#prediction').filter('p').text();
+        var label = innerHTML.substring(innerHTML.indexOf('Prediction:') + 'Prediction:'.length);
+        $('#prediction').html($('#prediction').html().replace(label, result['predicted_label']));
+      };
+
+      $.fn.serializeObject = function () {
+        var object = {};
+        $.each(this.serializeArray(), function () {
+          if (object[this.name] !== undefined) {
+            [object[this.name]].push(this.value || '');
+          } else {
+            object[this.name] = this.value || '';
+          }
+        });
+        return object;
+      };
+
+      function predict(data) {
+        var ENDPOINT = $('[name=url]').val();
+        return fetch(ENDPOINT, {
+          method: 'POST',
+          body: data,
+        })
+          .then((res) => res.json())
+          .catch(err => console.log('err', err));
+      };
+
+      $('#PredictButton').click(function (data = DATA) {
+        DATA = JSON.stringify($('form').serializeObject());
+        predict(DATA).then(body => updatePrediction(body));
+      });
+    });
+  </script>
+</head>
+<body>
+  <p>Bank Churn Prediction Service</p>
```

Comment:
https://aiclub.world/projects

i believe its customer churn

Reply:
@nishatalagala @sindhuPyxeda @swamisundar please confirm

Reply:
Ah, you're right. The website says customer churn. Changed to reflect.